public/index.html

```diff
@@ -3,11 +3,47 @@
   <head>
     <meta charset="utf-8">
     <title>Hoodie Account</title>
+    <link rel="stylesheet" href="/default/css/default.css">
     <script src="/hoodie/client.js"></script>
   </head>
   <body>
-    <p>
-       Open browser console and play with <code>hoodie.account</code>.
-    </p>
+
+    <div data-hide="false" class="generic-loader-wrap"></div>
+
+    <section data-hide="true" class="input-forms">
+      <h3>Sign In</h3><br>
```

Comment:
As the first header on the page, this should not be an h3.
Headings should not be used for decoration, they are important for content navigation.

So, the first heading on a page should be an H1.

Perhaps we can have an H1 heading of "Account" and then all the h3's can be h2 subheadings?